front/production_checks/checks/scrub_deleted_core_document_versions.ts

```diff
@@ -0,0 +1,196 @@
+import { Storage } from "@google-cloud/storage";
+import { createHash } from "blake3";
+import pino, { LoggerOptions } from "pino";
+import { Sequelize } from "sequelize";
+
+import { CheckFunction } from "../types/check";
+
+const { CORE_DATABASE_URI, SERVICE_ACCOUNT, DUST_DATA_SOURCES_BUCKET } =
+  process.env;
+
+export const scrubDeletedCoreDocumentVersionsCheck: CheckFunction = async (
+  checkName,
+  logger,
+  reportSuccess
+) => {
+  if (!CORE_DATABASE_URI) {
+    throw new Error("Env var CORE_DATABASE_URI is not defined");
+  }
+  if (!SERVICE_ACCOUNT) {
+    throw new Error("Env var SERVICE_ACCOUNT is not defined");
+  }
+  if (!DUST_DATA_SOURCES_BUCKET) {
+    throw new Error("Env var DUST_DATA_SOURCES_BUCKET is not defined");
+  }
+
+  const core_sequelize = new Sequelize(CORE_DATABASE_URI as string, {
+    logging: false,
+  });
+
+  const deletedDocumentsData = await core_sequelize.query(
+    `SELECT * FROM data_sources_documents WHERE status = 'deleted'`
+  );
```

Comment:
Side note: you will get back a `object[]` type if you add  `{ type: QueryTypes.SELECT }`:
```
.query(`...`, , { type: QueryTypes.SELECT })
```